### ISSUE TITLE
Disable model-missing-unicode (W5101) not working

### ISSUE THREAD

**Issue by muthuegrove** [landscapeio/pylint-django#63]
In pylintrc disabled the model-missing-unicode error, But its not working as we expected. We have integrated with prospector.

**In .pylintrc,**

disable=W5101 # W5101 - No **unicode** method on model

**## OutPut**
  Line: 177
    pylint: model-missing-unicode / No **unicode** method on model (Model1)
  Line: 184
    pylint: model-no-explicit-unicode / Model does not explicitly define **unicode** (Model2)
  Line: 190
    pylint: model-missing-unicode / No **unicode** method on model (Model3)
  Line: 199
    pylint: model-missing-unicode / No **unicode** method on model (Model4)

Note: Other disable options(cyclic-import, locally-disabled) is working fine

**Comment by atodorov** [comment]
I don't see W5101 inside the latest pylint sources but we do have it defined in several test cases and it doesn't cause an error:
```
test/input/external_drf_noerror_serializer.py:#  pylint: disable=C0111,W5101
test/input/external_psycopg2_noerror_postgres_fields.py:#  pylint: disable=C0111,W5101
test/input/func_noerror_uuid_field.py:#  pylint: disable=C0111,W5101
```

I'm closing this as worksfor me but feel free to open with a current reproducer.
